handler/handler_test.go

```diff
@@ -126,3 +128,239 @@ func TestGetAllEmployeesHandler_Return_invalid_status_code(t *testing.T) {
 	handlerInstance.GetAllEmployeesHandler(fakeContext)
 	assert.Equal(t, 400, responseRecorder.Code)
 }
+
+func TestCreateEmployeesHandler_SingleEmployee_valid_status_code(t *testing.T) {
+
+	var jsonPayload = `
+		{
+		"id": "100",
+		"first_name": "Test",
+		"last_name": "Tester",
+		"email": "tester@example.com"
+		}
+	`
+	var employeePayload model.Employee
+	json.Unmarshal([]byte(jsonPayload), &employeePayload)
+	body := bytes.NewBufferString(jsonPayload)
+	responseRecorder := httptest.NewRecorder()
+	fakeContext, _ := gin.CreateTestContext(responseRecorder)
+	fakeContext.Request = httptest.NewRequest("POST", "http://localhost:9090/employee/create", body)
+	fakeService := &handlerfakes.FakeServiceInterface{}
+	var fakeEmployees []model.Employee
+	var fakeEmployee model.Employee
+	fakeEmployee.ID = "100"
```

Comment:
Es ist schöner die Felder direkt im Objekt anzugeben, wie du das auch in den folgenden Tests gemacht hast.